test/inductor/test_cpu_select_algorithm.py

```diff
@@ -296,7 +278,10 @@ def forward(self, x):
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
         if (
             (
-                dtype == torch.bfloat16
+                (
+                    dtype == torch.bfloat16
```

Comment:
nit: you can merge these two with:

```python
(
    dtype in (torch.bfloat16, torch.float16)
    and torch.ops.mkldnn._is_mkldnn_fp16_supported()
)
```

Reply:
@XuehaiPan from what my debugging printouts, `torch.ops.mkldnn._is_mkldnn_fp16_supported()` seems always False, while `torch.ops.mkldnn._is_mkldnn_bf16_supported()` can be True or False.